Helper/Feed/Generator.php

```diff
@@ -584,9 +584,8 @@ protected function getUsedProductsMinPrice($product, $usedProds, $type)
 
         foreach ($usedProds as $child) {
             if ($child->getId() != $product->getId()) {
-                $price = $price === null ?
-                    $child->getPrice($type)
-                    : min($child->getPrice($type), $price);
+                $child_price = $child->getPriceInfo()->getPrice($type)->getAmount()->getBaseAmount();
```

Comment:
What is the difference between `$child->getPrice($type)` and `$child_price = $child->getPriceInfo()->getPrice($type)->getAmount()->getBaseAmount();`?

Reply:
I think `getBaseAmount` gets the price without the tax, where as `getPrice` returns the price with tax. Is that right @Jamstaz? 

$price_without_tax = $product->getPriceInfo()->getPrice($type)->getAmount()->getBaseAmount();
$price_with_tax = $product->getPriceInfo()->getPrice($type)->getAmount();